### PR TITLE
Create metadata table

### PR DESCRIPTION
**Pull request by adilahiri**
#### What scientific question is your analysis addressing?

Create Table S1 for future manuscript to contain the histologies file along with a README

#### What was your approach?
Adapted code from:
https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/tables/output_tables.Rmd

#### What GitHub issue does your pull request address?
[Issue#459
](https://github.com/PediatricOpenTargets/ticket-tracker/issues/459)

#### Which areas should receive a particularly close look?
Please review [TableS1-histologies.xlsx](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/blob/create-metadata-table/tables/results/TableS1-histologies.xlsx) and the script [output_tables.nb.html](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/blob/create-metadata-table/tables/output_tables.nb.html)

#### Is there anything that you want to discuss further?
NA

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes

#### What types of results are included (e.g., table, figure)?
Table

#### Reproducibility Checklist

- [ ] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

- [ X] This analysis module has a `README` and it is up to date.
- [ ] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [ X] The analytical code is documented and contains comments.

### CODE REVIEW

**Review by jharenza — CHANGES_REQUESTED** [review]
Hi @adilahiri thanks for starting this. Can you make sure all columns from v11 are represented in the readme? Eg - the methylation information. Thanks!

**Comment by adilahiri** [comment]
@jharenza thank you for pointing out the missing methylation columns. I have added them into the README now. 

Following are the extra columns that have been added to README. I have added the description based on my understanding, but they can be improved especially for the classification scores by specifying the type of score (accuracy, F-1, roc_auc_score etc). 

| Histology column| Definition| 
| :---         |     :---:      |      
| cns_methylation_class  | methylation-based CNS tumour classes  | 
| cns_methylation_class_score   | class score | 
| cns_methylation_subclass   | methylation-based CNS tumour sub-classes      | 
| cns_methylation_subclass_score   | subclass score     | 
| gtex_group   | Tissue Type| 
| gtex_subgroup   | Tissue Subtype | 
| RF_purity_ABSOLUTE   | purity score based on ABSOLUTE dataset | 
| RF_purity_ESTIMATE   | purity score based on ESTIMATE dataset | 
| RF_purity_LUMP   |purity score based on LUMP dataset |

File: tables/output_tables.Rmd [hunk 1]
```diff
@@ -0,0 +1,118 @@
+---
+title: "Tables output for manuscript"
+author: "Aditya Lahiri,Eric Wafula and Jo Lynne Rokita"
```
Comment (jharenza) [inline tables/output_tables.Rmd]:
```suggestion
author: "Aditya Lahiri"
```

File: tables/output_tables.Rmd [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+"OS_status","Overall survival status",paste(unique(histology_df$OS_status), collapse = "; "),
+"pathology_diagnosis","Reported and/or harmonized patient diagnosis from pathology reports","text",
+"pathology_free_text_diagnosis","Free text patient diagnosis from pathology reports","text",
+"PFS_days","Progression-free survival in days","numeric",
+"primary_site","Bodily site(s) from which specimen was derived","text",
+"race","Patient reported race",paste(unique(histology_df$race), collapse = "; "),
+"reported_gender","Patient reported gender",paste(unique(histology_df$reported_gender), collapse = "; "),
+"RF_purity_ABSOLUTE","purity score based on ABSOLUTE dataset","numeric",
+"RF_purity_ESTIMATE","purity score based on ESTIMATE dataset","numeric",
+"RF_purity_LUMP","purity score based on LUMP dataset","numeric",
```
Comment (jharenza) [inline tables/output_tables.Rmd]:
```suggestion
"RF_purity_ABSOLUTE","tumor purity fraction estimated using ABSOLUTE","numeric",
"RF_purity_ESTIMATE","tumor purity fraction estimated using ESTIMATE dataset","numeric",
"RF_purity_LUMP","tumor purity fraction estimated using LUMP","numeric",
```

File: tables/output_tables.Rmd [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+"age_at_diagnosis_days","Patient age at diagnosis in days","numeric",
+"age_last_update_days","Patient age at the last clinical event/update in days","numeric",
+"aliquot_id","External aliquot identifier","alphanumeric",
+"broad_histology","Broad WHO 2016 classification of cancer type",paste(unique(histology_df$broad_histology), collapse = "; "),
+"cancer_group","Harmonized cancer groupings for plots",paste(unique(histology_df$cancer_group), collapse = "; "),
+"cancer_predispositions","Reported cancer predisposition syndromes",paste(unique(histology_df$cancer_predispositions), collapse = "; "),
+"cns_methylation_class","methylation-based CNS tumour classes",paste(unique(histology_df$cns_methylation_class), collapse = "; "),
+"cns_methylation_class_score","class score","numeric",
+"cns_methylation_subclass","methylation-based CNS tumour sub-classes",paste(unique(histology_df$cns_methylation_subclass), collapse = "; "),
+"cns_methylation_subclass_score","subclass score","numeric",
```
Comment (jharenza) [inline tables/output_tables.Rmd]:
```suggestion
"cns_methylation_class","methylation-based CNS tumor class",paste(unique(histology_df$cns_methylation_class), collapse = "; "),
"cns_methylation_class_score","methylation-based CNS tumor class score","numeric",
"cns_methylation_subclass","methylation-based CNS tumor sub-class",paste(unique(histology_df$cns_methylation_subclass), collapse = "; "),
"cns_methylation_subclass_score","methylation-based tumor subclass score","numeric",
```

File: tables/run-manuscript-tables.sh [hunk 4]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Module authors: Aditya Lahiri,Eric Wafula and Jo Lynne Rokita
```
Comment (jharenza) [inline tables/run-manuscript-tables.sh]:
```suggestion
# Module author: Aditya Lahiri
```

**Review by jharenza — APPROVED** [review]
few comments, I will commit, rerun, and approve this so we can hand off to NCI

**Comment by jharenza** [comment]
I will merge this without checks since it doesn't run in GA

**Pull request merged**